brickstrap-unshare-harder.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
Those concerns about default Debian is why I kept it separate. It's meant as a fallback for the user to be able to recover easily if they're willing to accept a package and the 'fiddling'. 

The idea is the user tries brickstrap.sh: it works great; it doesn't they can use this as a fallback. Also with this it is simpler to ensure correctness in a sense, because I don't know exactly what else brickstrap.sh needs to trap and if there are edge-cases (which are easy to fix in this wrapper script if they appear, perhaps not so much in brickstrap.sh)...